packages/react/tabs/src/Tabs.stories.tsx

```diff
@@ -213,6 +277,38 @@ export const Chromatic = () => (
       </Tabs.Root>
     </DirectionProvider>
 
+    <h1>Animated</h1>
```

Comment:
Bit weird to see an animated story in chromatic but the long duration is to catch via diff when there's a simple regression in mount behaviour. Under normal conditions it should obviously not be animating at all.